examples/sensor-websocket/src/components/CredentialsModal.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from "react";
+import { Col, Form, Modal } from "react-bootstrap";
+
+class CredentialsModal extends Component {
```

Comment:
when extending react components you must make sure to init the internal component too
```
class MyClass extends React.Component {
  constructor(props) {
    super(props);

    // my init here
  }

  render() {
  }
}
```